protocol/xmrmaker/event.go

```diff
@@ -193,7 +193,9 @@ func (s *swapState) handleEvent(event Event) {
 		err := s.handleEventETHLocked(e)
 		if err != nil {
 			e.errCh <- fmt.Errorf("failed to handle EventETHLocked: %w", err)
-			return
+			if !s.fundsLocked {
```

Comment:
Is this variable only modified by a single go process?

Reply:
yep, all the swapState functions are accessed by one go process at a time.